### DISCUSSION TITLE
Does the VertexBuffer have a release API in filament?

### DISCUSSION THREAD

**Discussion by Haidiye00** [google/filament#7045] [Q&A] [answered]
in opengl,we can use "glDeleteBuffers(1, &vbo);" to release vbo,so in filament how to release VertexBuffer

**Comment by bejado** [comment]
Yes, see [`Engine::destroy(VertexBuffer*)`](https://github.com/google/filament/blob/96cccc83c6cd4c87ea658a09cac331b7439405ab/filament/include/filament/Engine.h#L650). We follow this pattern with other Filament objects too.

**Comment by Haidiye00** [comment, marked as answer]
thanks
